docs/advanced/input_files/input-main.md

```diff
@@ -3743,3 +3752,63 @@ These variables are used to control the usage of PEXSI (Pole Expansion and Selec
 - **Default**: 1e-10
 
 [back to top](#full-list-of-input-keywords)
+
+## Beyond DFT
+
+These parameters are used to solve the excited states using. e.g. LR-TDDFT.
+
+### xc_kernel
+
+- **Type**: String
+- **Description**: The exchange-correlation kernel used in the calculation. 
+Currently supported: `RPA`, `LDA`, `PBE`, `HSE`, `HF`.
+- **Default**: LDA
+
+### lr_solver
+
+- **Type**: String
+- **Description**: The method to solve the Casida equation in LR-TDDFT.
+  - `dav`: Construct $AX$ and diagonalize the Hamiltonian matrix iteratively with Davidson algorithm.
+  - `lapack`: Construct the full $A$ matrix and directly diagonalize with LAPACK.
+  - `spectrum`: Caluclate absorption spectrum only without solving Casida equation. The `OUT.${suffix}/` directory should contain the
+  files for LR-TDDFT eigenstates and eigenvalues, i.e. `Excitation_Energy.dat` and `Excitation_Amplitude_${processor_rank}.dat`
+   output by setting `out_wfc_lr` to true.
+- **Default**: dav
+
+### lr_thr
+
+- **Type**: Real
+- **Description**: The convergence threshold of iterative diagonalization fo LR-TDDFT.
+- **Default**: 1e-6
+
+### nvirt
+
+- **Type**: Integer
+- **Description**: The number of virtual orbitals used in the LR-TDDFT calculation.
+- **Default**: 1
+
+### nstates
```

Comment:
this name is too general, I suggest changing 'nstates' to 'lr_nstates' or something more specific